instrumentation/p6spy/src/main/java/brave/p6spy/TracingJdbcEventListener.java

```diff
@@ -69,9 +74,17 @@
   void parseServerAddress(Connection connection, Span span) {
     try {
       URI url = URI.create(connection.getMetaData().getURL().substring(5)); // strip "jdbc:"
+      String actualServiceName = remoteServiceName;
+      final Matcher matcher = URL_SERVICE_NAME_FINDER.matcher(url.toString());
```

Comment:
one other thing.. in our mysql interceptor, it is this:

```java
      String remoteServiceName = connection.getProperties().getProperty("zipkinServiceName");
```

is there a way to avoid manual parsing?

Reply:
I will check that.

Reply:
Actually `getProperties` is part of the internal MySQL API and not jdbc.